lib/plugins/bodyParser.js

```diff
@@ -160,25 +160,34 @@ function bodyParser(options) {
         }
 
         var parser;
-        var type = req.contentType().toLowerCase();
+        //var type = req.contentType().toLowerCase();
+        var type = req
+            .contentType()
```

Comment:
`req.contentType()` should already support handling `;`, did you find out otherwise? 

re: https://github.com/restify/node-restify/blob/master/lib/request.js#L210